docs/attributes-registry/csi.md

```diff
@@ -0,0 +1,20 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# CSI
+
+## Container Storage Interface (CSI) Attributes
+
+Attributes related to the Container Storage Interface (CSI).
+
+| Attribute         | Type   | Description                                          | Examples                                                      | Stability                                                        |
+| ----------------- | ------ | ---------------------------------------------------- | ------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `csi.plugin.name` | string | The name of the CSI plugin used by the volume. [1]   | `pd.csi.storage.gke.io`                                       | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Are these standalone attributes still valid 🤔 ? I thought these would only go under the `container.*` namespace, or I miss something here?

Reply:
Good catch... I think there's a hole in the generator process that should be removing files prior to generate to catch stragglers like this. I removed it by hand for now.

Reply:
fyi @lmolkova @trisch-me

Reply:
great catch! It seems like a merge issue after #1345 (a lot of files were moved around)